CONTRIBUTING.rst

```diff
@@ -1,4 +1,72 @@
-Contributing to ``pycodestyle``
+Contributing to pycodestyle
 ===============================
 
-Please see the `developer notes <https://pycodestyle.readthedocs.io/en/latest/developer.html>`_
+When contributing to pycodestyle, please observe our `Code of Conduct`_.
+
+Step 1: Forking pycodestyle for editing
+---------------------------------------
+
+Fork the pycodestyle repository on github. This will add
```

Comment:
Well, given "GitHub" is it's actual name (as trademarked and branded) that makes sense to me. It's also how I prefer to spell it.